docs/user/user/pings/data-control-plane/experimenter-configuration.md

```diff
@@ -0,0 +1,23 @@
+# Experimenter Configuration
+
+The structure of this configuration is a key-value collection with the full metric identification of the Glean metric serving as the key in the format <metric_category.metric_name>.
+
+The values of the key-value pair are booleans which represent whether the metric is enabled (`true`) or not (`false`).
+
+In the example below `gleanMetricConfiguration` is the name of the variable defined in the Nimbus feature.
+
+This configuration would be what is entered into the branch configuration setup in Experimenter when defining an experiment or rollout.
```

Comment:
This appears to be describing only the `metrics_enabled` substructure of `gleanMetricConfiguration`, not the entirety (which I'm given to understand has both a `metrics_enabled` and a `ping_enabled` portion)

Reply:
Either way the configuration is entered into the branch payload in Experimenter. Is there some specific wording here that would help to make this more clear to you?

Reply:
"the full metric identification of the Glean metric serving as the key...." is metrics stuff, not pings stuff, right?